plugins/modules/aci_l3out_floating_svi.py

```diff
@@ -0,0 +1,489 @@
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_floating_svi
+short_description: Manage Layer 3 Outside (L3Out) interfaces (l3ext:RsPathL3OutAtt)
+description:
+- Manage L3Out interfaces on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+    required: true
+  interface_profile:
+    description:
+    - Name of the interface profile.
+    type: str
+    aliases: [ interface_profile_name, logical_interface ]
+    required: true
+  pod_id:
+    description:
+    - Pod to build the interface on.
+    type: str
+  node_id:
+    description:
+    - Node to build the interface on for Port-channels and single ports.
+    type: str
+  encap:
+    description:
+    - Encapsulation on the interface (e.g. "vlan-500")
+    type: str
+  encap_scope:
+    description:
+    - Encapsulation scope.
+    choices: [ vrf, local ]
+    type: str
+  address:
+    description:
+    - IP address.
+    type: str
+    aliases: [ addr, ip_address ]
+  mac_address:
+    description:
+    - The MAC address option of the interface.
+    type: str
+  link_local_address:
+    description:
+    - The link local address option of the interface.
+    type: str
+  mtu:
+    description:
+    - Interface MTU.
+    type: str
+  ipv6_dad:
+    description:
+    - IPv6 DAD feature.
+    type: str
+    choices: [ enabled, disabled]
+  mode:
+    description:
+    - Interface mode, only used if instance_type is ext-svi
+    type: str
+    choices: [ regular, native, untagged ]
+  dscp:
+    description:
+    - The target Differentiated Service (DSCP) value.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
+    aliases: [ target_dscp ]
+  external_bridge_group_profile:
+    description:
+    - The external bridge group profile.
+    - Pass "" as the value to remove an existing external bridge group profile (See Examples).
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  auto_state:
+    description:
+    - SVI auto state.
+    type: str
+    choices: [ enabled, disabled ]
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+seealso:
+- module: aci_l3out
```

Comment:
I added tags for modules it is immediately dependent on. When a user sees that immediate module the it will have the respective info.

Reply:
Not everywhere. See: https://github.com/CiscoDevNet/ansible-aci/blob/0dcf72cdf86875ec818e1790797274175d2786aa/plugins/modules/aci_l3out_logical_interface_profile.py
However, it might be useful and easier for the user to have more info. I'll go ahead and add those.